### PR TITLE
Update beta v0.5.17 in readme status

### PR DESCRIPTION
**Pull request by pizofreude**
chore: update beta v0.5.17 in readme under status header

### CODE REVIEW

**Comment by pizofreude** [comment]
@behics this is PR for [issue 503](https://github.com/languageXchange/languageXchange/issues/503)
Kindly check.

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ languageXchange is an application built with Angular and Ionic that helps users
 
 ## Status
 
-This project is currently in Beta phase v0.X.x For more information, please refer to the project's GitHub repository.
+This project is currently in Beta phase v0.5.17 For more information, please refer to the project's GitHub repository.
```
Comment (behics) [inline README.md:31]: Can you please change with `v0.5.x`

why: _because no need to change every minor update_

**Review by behics — CHANGES_REQUESTED** [review]
Thank you for your contributrion

**Review by behics — APPROVED** [review]
Done!

**Pull request merged**